src/main/java/org/syncninja/model/committree/CommitNode.java

```diff
@@ -1,10 +1,12 @@
 package org.syncninja.model.committree;
 
 import org.neo4j.ogm.annotation.NodeEntity;
+import org.syncninja.dto.FileStatusEnum;
 import org.syncninja.model.SyncNode;
 
 @NodeEntity
 public abstract class CommitNode extends SyncNode {
+    private FileStatusEnum StatusEnum;
```

Comment:
Can you please fix the name so that it does not start with capital `S` ex: `statusEnum`?